templates/openwisp2/settings.py

```diff
@@ -338,7 +341,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.9/howto/static-files/
 
-STATICFILES_DIRS = [os.path.join(BASE_DIR, 'static-custom')]
+STATICFILES_DIRS = [os.path.join(BASE_DIR, 'ow2_static')]
```

Comment:
I didn't mean to change this, this was fine (although for consistency we can use `static_custom`, I meant the directory used in the playbook, please see https://github.com/openwisp/ansible-openwisp-wifi-login-pages/#deploying-custom-static-content